src/components/ui/IconButton.js

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+const StyledIconButton = styled.button`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: ${({ size }) => size}px;
+  height: ${({ size }) => size}px;
+  padding: 0;
+  background-color: white;
+  border-radius: 50%;
+  border: 2px solid
+    ${({ raised, theme }) => (raised ? 'white' : theme.secondaryBackground)};
+  ${({ raised, theme }) =>
+    raised &&
+    `box-shadow: 0 0 8px 1px ${theme.shadow};
+	`}
+
+  &:focus {
+    outline: none;
+  }
+`
+
+const RaisedIconButtonContainer = styled.span`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  width: ${({ size }) => size}px;
+  height: ${({ size }) => size}px;
+  border: 2px solid;
+  border-color: orange;
+`
```

Comment:
Would be nice to look into ways to achieve multiple border without an extra container: https://css-tricks.com/snippets/css/multiple-borders/

Don't worry about this now though!